src/Microsoft.Graph.Core/Models/UploadSession.cs

```diff
@@ -42,7 +42,9 @@ internal class UploadSession : IUploadSession
             return new Dictionary<string, Action<IParseNode>> 
```

Comment:
```suggestion
            return new Dictionary<string, Action<IParseNode>> (StringComparer.OrdinalIgnoreCase)
```

Reply:
Thanks @baywet This was a much better suggestion :)